### ISSUE TITLE
Does regex checking account for Filter Lists?

### ISSUE THREAD

**Issue by Marsfan** [yubiuser/pihole_adlist_tool#32]
When using the regex checking feature, does the list of blocked domains include the fact that they might also be blocked by a filter list, or is the regex checker independent of the filter list checker? 

I am trying to determine if I need the regex configurations that I have applied.

**Comment by yubiuser** [comment]
This has been merged into master.
